### PR TITLE
[geometry-1] Add DOMPoint test for Editor’s Draft.

### PR DESCRIPTION
**Pull request by hwanseung**
https://drafts.fxtf.org/geometry/#dompointreadonly
https://drafts.fxtf.org/geometry/#dompoint

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/csswg-test/1255)

### CODE REVIEW

**Comment by hwanseung** [comment]
@zcorpan could you reivew this patch?

File: geometry-1/DOMPoint-002.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            var p = new DOMPointReadOnly(0, 0, 0, 1);
+            p.x = undefined;
+            p.y = undefined;
+            p.z = undefined;
+            p.w = undefined;
+            checkDOMPoint(p, {x:0, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Attributes');
+
+        function checkDOMPoint(p, exp) {
+            assert_equals(p.x, exp.x, "Expected value for x is " + exp.x);
```
Comment (zcorpan) [inline geometry-1/DOMPoint-002.html]: The generated message for `assert_equals` already says what the expected value is. Just need to say which property is being asserted.

File: geometry-1/DOMPoint-002.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            var p = new DOMPoint(0, 0, 0, 1);
+            p.x = NaN;
+            p.y = Number.POSITIVE_INFINITY;
+            p.z = Number.NEGATIVE_INFINITY;
+            p.w = Infinity;
+            checkDOMPoint(p, {x:NaN, y:Infinity, z:-Infinity, w:Infinity});
+        },'test DOMPoint Attributes NaN Infinity');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly(), {x:0, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Constructor0');
```
Comment (zcorpan) [inline geometry-1/DOMPoint-002.html]: Can you describe the test in the test name and avoid incrementing a number? e.g. `DOMPointReadOnly constructor no args`

**Comment by zcorpan** [comment]
Can you rebase?

**Comment by hwanseung** [comment]
@zcorpan 
i fixed it. could you review again?

**Review by zcorpan — APPROVED** [review]

**Pull request merged**